### PR TITLE
Vehicles - Add Theseus Cargo / Transport Vans

### PR DESCRIPTION
**Pull request by TyroneMF**
- Adds a Theseus Branded Cargo/Transport van, randomly selects texture on spawn.
- Separated `CfgVehicles` out into separate files.
- Moved Polaris Macro to `polaris.hpp` from Script component.
- ~~Adds the "Verstappen Karts" texture to the TextureSources of the civilian Cargo Van variant.~~
- New base classes for Transport & Cargo Vans

### CODE REVIEW

File: addons/vehicles/vehicles/van.hpp [hunk 1]
```diff
@@ -0,0 +1,186 @@
+class Van_02_base_F;
+class Van_02_vehicle_base_F: Van_02_base_F {
+    class EventHandlers;
+
+    class TextureSources {
+        class Verstappen {
```
Comment (jonpas) [inline addons/vehicles/vehicles/van.hpp:6]: This really doesn't feel like it belongs into Theseus Services.

Comment (TyroneMF) [reply addons/vehicles/vehicles/van.hpp:6]: No idea where else to put it tbh.

Comment (jonpas) [reply addons/vehicles/vehicles/van.hpp:6]: Maybe an optional? If not GT Armory.

It's not Theseus Services that should provide it I think.

Comment (TyroneMF) [reply addons/vehicles/vehicles/van.hpp]: And it's gone

**Review by jonpas — APPROVED** [review]

**Pull request merged**